University-advisor/Forms/MainForm.cs

```diff
@@ -164,34 +175,101 @@ private void ChangeEmail_Click(object sender, EventArgs e)
                 {
                     errorMessageEmail.Text = "Emails doesn't match.";
                     errorMessageEmail.Show();
+                    Logger.Log("Emails doesn't match.");
+
                 }
             }
         }
 
         private void ChangeUniversity_Click(object sender, EventArgs e)
         {
             errorMessageUniversity.Hide();
-            if (!universityBox.SelectedItem.Equals("Current university in DB"))
+            string sqlGetCurrentUniversity = "select universities.name from universities, users where universities.universityId = users.universityid and users.username = '" + currentUser + "';";
+            ArrayList universityIdFromDB = SqlDriver.Fetch(sqlGetCurrentUniversity);
+            string currentUniversity = ((Dictionary<string, object>)universityIdFromDB[0])["name"].ToString();
+
+            if (!universityBox.SelectedItem.Equals(currentUniversity))
             {
-                //Change university in DB
-            } else
+                string sqlGetNewUniversityID = "select universityid from universities where name ='" + universityBox.SelectedItem+"';";
+                ArrayList newUniversityIdFromDB = SqlDriver.Fetch(sqlGetNewUniversityID);
+                string newUniversityId = ((Dictionary<string, object>)newUniversityIdFromDB[0])["universityId"].ToString();
+
+                string sqlUpdateUniversityID = "update users set universityid =" + newUniversityId + " where username ='" + currentUser + "';";
+                try
+                {
+                    if (SqlDriver.Execute(sqlUpdateUniversityID))
+                    {
+                        MessageBox.Show("University updated successfully.");
+                        Logger.Log("University updated successfully.");
+                    }
+                    else
+                    {
+                        MessageBox.Show("Error changing university");
+                        Logger.Log("Error changing university");
+                    }
+                }
+                catch (Exception ex)
+                {
+                    Logger.Log(ex.Message);
+                }
+
+            }
+            else
             {
                 errorMessageUniversity.Text = "New university cannot be the same as old one.";
                 errorMessageUniversity.Show();
+                Logger.Log("New university cannot be the same as old one.");
             }
         }
 
         private void ChangeStatus_Click(object sender, EventArgs e)
         {
             errorMessageStatus.Hide();
-            if (!statusBox.SelectedItem.Equals("Current status in DB"))
+
+            string sqlGetCurrentStatus = "SELECT status from users where username='" + currentUser + "';";
+            ArrayList statusFromDB = SqlDriver.Fetch(sqlGetCurrentStatus);
+            string currentStatus = ((Dictionary<string, object>)statusFromDB[0])["status"].ToString();
```

Comment:
In this lines use var declaration.

Reply:
Is var really that better? In this case it is easier to understand what object would be returned. var obscures the actual variable type. If the initializer doesn’t return a clearly defined type then you may not be able to tell the variable’s type.

Reply:
.NET 3.0 and higher offers an option to use this kind of declaration. Why you afraid to use it? If you are interested in a type you can just place cursor on this variable and you will find out. But why is it better:  "When the compiler encounters fields during code compilation, it records each field's type before processing any expressions associated with it." 
So basically compiler does not have to think about types which is definitely better. More on that in Microsoft documentation: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables

Reply:
Thanks for taking your time and explaining the usage of var. I will try to use it more often, but this time I will leave it as it is